internal/network/ports_test.go

```diff
@@ -557,4 +563,163 @@ var _ = Describe("Network Ports", func() {
 			})
 		})
 	})
+
+	Describe("GetProxyPorts", func() {
+		var originalEnvVars map[string]string
+
+		BeforeEach(func() {
+			// Save original environment variables
+			originalEnvVars = make(map[string]string)
+			for _, envVar := range []string{"HTTP_PROXY", "HTTPS_PROXY", "http_proxy", "https_proxy", "NO_PROXY", "no_proxy"} {
```

Comment:
I would use util.GetProxyEnvVars() here.

Reply:
Added in a constant for this and updated the code accordingly.